freqtrade/optimize/backtesting.py

```diff
@@ -173,6 +173,7 @@ def __init__(self, config: Config, exchange: Exchange | None = None) -> None:
         self.disable_database_use()
         self.init_backtest_detail()
         self.pairlists = PairListManager(self.exchange, self.config, self.dataprovider)
+        self.dinamic_pairlist = False
```

Comment:
```suggestion
        self.dynamic_pairlist = False
```

Obviously applies to every other line using this variable, too